charts/prometheus-mysql-exporter/values.yaml

```
@@ -42,6 +42,16 @@ serviceMonitor:
   metricRelabelings: []
   # Set relabel_configs as per https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
   relabelings: []
+  # Enable multi target scraping.
+  multipleTarget: false
+  targets: []
```

can we haver that like this please?
seems the readme above already use it like this too?

```suggestion
  multipleTarget:
    enabled: false
    targets: []
```

---

This should be corrected